src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -43,6 +43,7 @@ class ImageBaseViewer extends BaseViewer {
             return;
         }
 
+        ImageBaseViewer.setOriginalImageSize(this.imageEl);
```

Comment:
setOriginalImageSize is implemented as a static method right now. Is there any preference for avoiding static methods? I know that only one Viewer will actually be instantiated so there will not be much of a difference in our use case but I am advocating for the use of static methods to avoid the creation of monolithic utility files.